Assets/LeapMotion/Modules/InteractionEngine/Scripts/InteractionController.cs

```diff
@@ -1255,6 +1255,9 @@ public abstract class InteractionController : MonoBehaviour,
 
         if (_softContactCollisions.Count > 0) {
           _disableSoftContactEnqueued = false;
+          if (_delayedDisableSoftContactCoroutine != null) {
+            manager.StopCoroutine(_delayedDisableSoftContactCoroutine);
```

Comment:
Suuuper interesting. Is _this_ the line that seemingly fixes the soft contact weirdness? What's your understanding of how it actually fixes the issue?

Reply:
Well, the delayed soft contact disabling routine would get spuriously called between 1 and 3 times before this; after this it seems to only ever execute once.

Reply:
As far as the root cause of the issue... I don't fully know.   To @Amarcolina 's question, I had been testing the soft contact by disabling the disable call to it, but it seems like the bugs I was getting only appeared when I disabled Soft Contact's ability to disable itself (and go away when it's allowed to enable and disable itself...)

Reply:
(One hypothesis is that the velocity queries get corrupted when the soft contact colliders turn to trigger colliders)

Reply:
What does this piece of code _do_ exactly?  It seems to be called from within `fixedUpdateSoftContact`, does that mean it's being triggered every frame?